docker-images/kafka/cruise-control-scripts/cruise_control_config_generator.sh

```diff
@@ -36,5 +36,6 @@ broker.metric.sample.store.topic=__KafkaCruiseControlModelTrainingSamples
 capacity.config.file=$CC_CAPACITY_FILE
 cluster.configs.file=$CC_CLUSTER_CONFIG_FILE
 webserver.accesslog.path=/tmp/access.log
+webserver.http.address=$(hostname)
```

Comment:
this is the address on which CC is listening for incoming HTTP requests right? Maybe we want something like the Kafka Connect REST API which is `$(hostname -I)`. Anyway we are hitting the CC REST API through the related service, right?

Reply:
Yes and yes, I will update

Reply:
Actually, not for the `$(hostname -I)`, The `webserver.http.address` field is meant to only take one address [1]. `$(hostname -I)` has the possibility of listing more than one IP address as the `-I` argument causes the command to "enumerates all configured addresses on all  network  interfaces"

[1] https://github.com/linkedin/cruise-control/blob/master/cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/KafkaCruiseControlMain.java#L66